src/c-api.adoc

```diff
@@ -746,13 +746,18 @@ statements, including both RISC-V specific and common operand constraints.
 |K                  |5-bit unsigned immediate integer operand     |
 |J                  |Zero integer immediate operand               |
 |s                  |symbol or label reference with a constant offset |
+|cr                 |RVC general purpose register (`x8`-`x15`)  |
+|cf                 |RVC floating point register (`f8`-`f15`)   |
```

Comment:
Should we treat this as `cr` (`x8`-`x15`) for `Zfinx` or just reject that if `Zfinx`?

I prefer treat treat as `cr` for `Zfinx` so that user can  easier to write `Zfinx`-portable code

Reply:
Turns out that Craig already implemented the latter in LLVM: https://github.com/llvm/llvm-project/commit/1bc1a79a65a93a0224b5e5f69584219f9981bd23 (which I am happy with the direction of)

I'm going to push an update noting that "floating point register" means a register suitable for passing a floating point value, so with the `Z*inx` architectures, this means a GPR.